airflow/contrib/auth/backends/google_auth.py

```diff
@@ -112,8 +112,7 @@ def login(self, request):
         log.debug('Redirecting user to Google login')
         return self.google_oauth.authorize(callback=url_for(
             'google_oauth_callback',
-            _external=True,
-            _scheme='https'),
```

Comment:
We can probably re-enable this for note. `https://etl.doordash.com` does work, but I need to test it. For now, this is safer.